src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,79 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { FC } from 'react';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number | string) => void,
+};
+
+export const Pagination: FC<Props> = (props) => {
+  const {
+    total,
+    perPage,
+    currentPage,
+    onPageChange,
+  } = props;
+
+  const lastPage = Math.ceil(total / perPage);
+  const numbersOfPages = getNumbers(1, lastPage);
+
+  return (
+    <ul className="pagination">
+      <li className={classNames('page-item', {
+        disabled: currentPage === 1,
+      })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={() => {
+            if (currentPage !== 1) {
+              onPageChange('prev');
+            }
+          }}
```

Comment:
redundant checking
button is disabled, when this condition isn't met
```suggestion
          onClick={() => onPageChange('prev')}
```